Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -106,6 +110,10 @@ def __init__(self, struct_or_class, cf_type, namespace, name, parent_class_name,
                         self.from_cf_method = value
                     if key == 'ForwardDeclaration':
                         self.forward_declaration = value
+                    if key == 'WebKitSecureCodingClass':
```

Comment:
All the possible attributes currently have comment documentation at the top of this file.  Please add such documentation for all new attributes.

Reply:
Will do.